lib/phoenix/live_dashboard/live/table_helpers.ex

```diff
@@ -31,10 +31,25 @@ defmodule Phoenix.LiveDashboard.TableHelpers do
 
       %{} ->
         params = %{params | sort_dir: sort_dir, sort_by: sort_by}
-        live_patch(body, to: live_dashboard_path(socket, live_action, node, [], params))
+        live_patch(body, to: live_dashboard_path(socket, action, node, [], params))
     end
   end
 
+  def live_modal(socket, component, opts) do
+    path = Keyword.fetch!(opts, :return_to)
+    modal_opts = [id: :modal, return_to: path, component: component, opts: opts]
+    live_component(socket, Phoenix.LiveDashboard.ModalComponent, modal_opts)
+  end
+
+  def encode_pid(pid) do
+    pid
+    |> :erlang.pid_to_list()
+    |> tl()
+    |> Enum.drop(-1)
+    |> List.to_string()
+  end
+  def decode_pid(list_pid), do: :erlang.list_to_pid([?< | String.to_charlist(list_pid)] ++ [?>])
```

Comment:
I don't think it belongs in this module. This function is specific to PID stuff and table helpers is meant to be shared across ets, pid, sockets, etc.

Reply:
Luckily these functions only seem to be used in the ProcessesLive, so moving them should be easy.